code/modules/research/techweb/all_nodes.dm

```diff
@@ -1256,17 +1256,15 @@
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 
-/datum/techweb_node/combat_cyber_implants
+/datum/techweb_node/combat_cyber_implants // SKYRAT EDIT - REMOVED: X-rays+Thermals from here
 	id = "combat_cyber_implants"
 	display_name = "Combat Cybernetic Implants"
 	description = "Military grade combat implants to improve performance."
 	prereq_ids = list("adv_cyber_implants","weaponry","NVGtech","high_efficiency")
 	design_ids = list(
```

Comment:
Move the Skyrat edit to like, the line before this one or something like that.

Reply:
I literally meant this line here.

Reply:
👍